src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -26,6 +26,7 @@
     private ExportFormat format;
     private CypherFormat cypherFormat;
     private final Map<String, Object> config;
+    private boolean writeDataTypes;
```

Comment:
togli questa proprietà ed usa **useTypes** che è già presente e viene usata ad esempio nella classe che esporta in graphML [ExportGraphMLTest](https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/39559c80aeb510d4dfe9adb0354354eb23a34604/src/test/java/apoc/export/graphml/ExportGraphMLTest.java#L100) vedi anche [qui](https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_graphml_import_export)